examples/tfengine/modules/team.hcl

```diff
@@ -468,6 +468,17 @@ template "additional_iam_members" {
           ]
         }
       }]
+      "service_account" = [{
+        resource_ids = [
+          "runner@{{.prefix}}-{{.env}}-apps.iam.gserviceaccount.com",
+        ]
+        bindings = {
+          "roles/iam.serviceAccountKeyAdmin" = [
+            "group:{{.prefix}}-team-admins@{{.domain}}"
```

Comment:
This group probably doesn't exist. You can create another service account in the team.hcl and use that here to assign the IAM role.

Reply:
got it